app/controllers/api/v2/fact_values_controller.rb

```diff
@@ -18,6 +18,11 @@ def index
         host_id = params[:host_id]
         @fact_values = FactValue.build_facts_hash(values, host_id)
       end
+
+      def setup_search_options
+        params[:search] ||= ""
+        params[:search] += " host = " + params[:host_id] if params[:host_id]
```

Comment:
Doesn't the original `Api::BaseController#setup_search_options` do exactly this, but in a slightly more generic way?

Reply:
Ah, I see